pkg/run/session/remove.go

```diff
@@ -31,8 +32,8 @@ func Remove(kubeClient client.Client, session *InternalSession) error {
 
 	if err := kubeClient.Get(context.Background(),
 		types.NamespacedName{
-			Namespace: session.Namespace,
-			Name:      session.Name,
+			Namespace: session.SessionNamespace,
```

Comment:
I suppose this fixes the issue with `gitops remove run` not removing the session that a user would expect it would, right? We should definitely call this fix out explicitly, at best by creating an issue and referring to it in this PR.

Reply:
No it's not related to any other issue. The field was renamed as the part of this PR only.
We have "session namespace" and the "namespace" which is from the --namespace flag.

This PR changed "namespace" field to "session namespace" field
and used the "namespace" field to hold the value from --namespace flag.